charts/ad2image/templates/configmap.yaml

```
@@ -1,9 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "ad2image-helm-chart.fullname" . }}
+  name: {{ include "common.fullname" . }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)